.github/workflows/common.yml

```diff
@@ -0,0 +1,35 @@
+name: common
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+env:
+  PACKAGEPATH: ${{ github.workspace }}/go/src/github.com/${{ github.repository }}
+  GOLANGCI_LINT_VERSION: v1.23.7
+jobs:
+  test:
+    strategy:
+      matrix:
+        go-version: [1.14.x]
+        platform: [ubuntu-latest, macos-latest]
```

Comment:
do we need to test in the same version lyft is running?

Reply:
Chatted offline, decided we will defer for the internal rollout.